meal-sharing/src/app/meals/page.js

```diff
@@ -0,0 +1,15 @@
+import MealList from "@/components/Meals/MealList";
+async function fetchMeals() {
+  try {
+    const apiResponse = await fetch("http://localhost:3001/api/meals");
+    const data = await apiResponse.json();
+    return data;
+  } catch (error) {
+    throw new Error(error);
+  }
```

Comment:
This code (catching an error) is redundant in this case: the same error will be thrown further anyway. The good way to catch an error would be to handle it properly: show an error message or other special state. Right now it shows like this, which is not very user friendly

<img width="760" alt="image" src="https://github.com/user-attachments/assets/a290a145-45c7-4625-befc-bbbb0fab181a">

Reply:
Thank you for your inputs.I have now handled the error by displaying an error message.